accepted/global-json-updates.md

```diff
@@ -97,46 +92,55 @@ The `global.json` schema to offers support for all of the identified scenarios t
   * `usePreview`
     * `true` or `false`
     * Default: `true` to match today's behavior.
-  * `rollForward`
-    * The `version` must be specified if `rollForward` is specified.
-    * To avoid some of the repetitiveness, I do not restate in these rules that version numbers below that specified are always ignored. 
-    * `patch`
+  * `rollForward` (The `version` must be specified if `rollForward` is specified.)
+    * `patch` **[[(I've gone back and forth on "patch" or "legacy" here)]]**
       * If the requested major, minor, feature band, and patch if found, use it.
       * Otherwise, use the highest patch within the specified major, minor and feature band.
+      * This recreates the previous SDK selection criteria.
     * `feature`
-      * If the requested major, minor, feature band, and patch is found, use it.
-      * Otherwise, if the requested major, minor and feature band is found, use the highest patch in that band.
-      * Otherwise, use the lowest higher feature band, and use the highest patch within that feature band.
+      * If the requested major, minor, feature band, is found, use the highest available patch in that band.
+      * Otherwise, use the lowest higher major/minor/feature band, use the highest patch within that feature band.
```

Comment:
I don't get the intent of the new language and this wasn't my take away from this morning's meeting.

If I'm a user that specifies that I would like to roll-forward on `feature`, then I would expect dotnet to **not** cross a major and minor version (i.e. the major and minor must match what's requested).  I would expect it to roll forward to the next available feature band and the latest patch within that feature band, as per previously describe.

Same comment can be repeated for the changes to the `minor` and `major` policies.

Reply:
I think it would help to use the the same language as https://github.com/dotnet/designs/blob/8fb85d0a79f33b1548e884a8ee5b85319b5946a1/accepted/runtime-binding.md#rollforward

My understanding is that we will have the same behavior, and the only change is making it four levels instead of three (feature added) and what appears like a patch is semver is actually (feature * 100) + patch. If there is another difference, then I have misunderstood something and that would change what I said in the meeting.

Reply:
I really struggled with understanding that language and how long it would make this document. But a) @peterhuene 's comment is correct, my language is incorrect at present and b) @nguerrera 's indicates that my language is not solving the clarity issue. I'll take another look.

Reply:
Thanks, Kathleen.  I just wanted to double check as that is how I implemented it.

I'm using your grid as basis for some automated tests, plus others covering `allowPrerelease`, parsing of the JSON, etc.

Currently if there is an error in the `global.json` I emit a detailed error message of what is wrong and print out a message that the latest installed SDK (with or without prereleases depending on how the resolver was invoked) will be used.  That's what we previously did for errors, although errors other than a parse error were completely silent by default (required tracing to diagnose).